templates/repo/release_tag_header.tmpl

```diff
@@ -0,0 +1,27 @@
+{{$canReadReleases := $.Permission.CanRead $.UnitTypeReleases}}
+{{$canReadCode := $.Permission.CanRead $.UnitTypeCode}}
+<div class="release-tag-header gt-df gt-ac">
+	<div class="release-tag-header-left gt-df gt-ac gt-f1">
+		{{if $canReadReleases}}
+			<h2 class="ui compact small menu header small-menu-items">
+				<a class="{{if .PageIsReleaseList}}active {{end}}item" href="{{.RepoLink}}/releases">{{.locale.Tr "repo.release.releases"}}</a>
+				{{if $canReadCode}}
+					<a class="{{if .PageIsTagList}}active {{end}}item" href="{{.RepoLink}}/tags">{{.locale.Tr "repo.release.tags"}}</a>
+				{{end}}
+			</h2>
+
+			{{if .EnableFeed}}
+				<a class="rss-icon" href="{{.RepoLink}}/{{if .PageIsTagList}}tags{{else}}releases{{end}}.rss" data-tooltip-content="{{.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</a>
+			{{end}}
+		{{else if $canReadCode}}
+			{{template "repo/sub_menu" .}}
```

Comment:
This is a full width menu. Literally, it shouldn't appear in `xxxx-left`.

The use of this template  is quite complex, that's why it's not suitable to move more code into it.

Reply:
Hmm I see. Seems a bit odd to render this here, I will remove it.

Reply:
For the record, rendering before this PR with `canReadReleases` false and `CanCreateRelease` true:

<img width="1247" alt="Screenshot 2023-05-08 at 21 53 10" src="https://user-images.githubusercontent.com/115237/236920526-0b954a46-6c80-4a78-b504-6910eb746557.png">

Reply:
Fixed, I removed the fallback header, don't see much point in rendering there.

Reply:
> Hmm I see. Seems a bit odd to render this here, I will remove it.

@HesterG you see, this "sub_menu" could be easily removed, as I said ..... you know more background, please help to describe and write comments.

Reply:
I haven't worked on this page or this template before, not sure what is the background here.

Reply:
Oh sorry, my bad 🤣 actually this one:

* #23465

@Zettat123 please help ~~~

Reply:
https://github.com/go-gitea/gitea/blob/023a048f52b5bf8c4b715285245a129f04e05a8c/templates/repo/sub_menu_release_tag.tmpl#L15-L16

This code is to show the `sub_menu` in the `Tags` tab  when a repo's release setting is disabled. I think we should keep the `sub_menu`, otherwise the `Tags` tab will look different from the `Commits` tab and `Branches` tab. 

![image](https://user-images.githubusercontent.com/15528715/237049849-c3d4b56f-72d7-4937-a997-8be4e02485f7.png)

![image](https://user-images.githubusercontent.com/15528715/237049928-2cc2d8a0-a519-4ab6-842f-683428f028e0.png)

![image](https://user-images.githubusercontent.com/15528715/237043261-ee566550-89e0-4f65-a7ce-07bf6333d377.png)

Reply:
> otherwise the Tags tab will look different from the Commits tab and Branches tab.

Hmm, but wo don't show that `repo/sub_menu` on other tabs like Wiki, Releases, Projects, so it's already not everywere.

Reply:
Oh, I think I didn't make it clear.
If a repo's release setting is **enabled**, the tag list should be in the `Release` tab and we don't need to show the `repo/sub_menu`.
If the release setting is **disabled**, we will no longer have the `Release` tab, so the tag list should be in the `Code` tab's `Tags` sub-tab. To have the same style as the `Commits` and `Branch` sub-tabs, we need to keep the `repo/sub_menu` in this case.

But I found another bug that the `Code` tab isn't active when the `Tags` sub-tab is selected.

![image](https://github.com/go-gitea/gitea/assets/15528715/e2995195-92ce-4193-be98-c6cb84b56696)
![image](https://github.com/go-gitea/gitea/assets/15528715/3b0ec372-bc60-472a-8c1e-0e58f20b1c83)

Reply:
Ah I see. I agree a page without any active tab does look odd in that case.

Reply:
Restored it. Don't wanna deal with the tab active issue now and I see it as very minor and unrelated.

<img width="1250" alt="Screenshot 2023-05-10 at 00 00 51" src="https://github.com/go-gitea/gitea/assets/115237/2c43c90a-47ba-4911-9e45-e902d54d68c9">